menu.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'app'
+class Main
+  def initialize
+    app = App.new
+    app.main_menu
+  end
+end
```

Comment:
- [ ] Let's think together, what about this `Main` class, what it actually does? and if we removed it, is anything gonna change in the app?
Kindly notice that in the `app.rb` file, you already have initialized the class `App` and called its methods the last thing in the file, so I guess you don't need a whole new class to just call it right?
Another thing, I don't see any use of this class `Menu` in any other file. 🙏

Reply:
This is the main entry point and a part of every Ruby project requirement so far, so what do you suggest please? As I know from our past projects, we've done this and never got in the way of the approval

Reply:
You are totally right, but please don't forget that (as you always do) the main file is created for the UI and displaying the choices of the app, and since you already done that in the app.rb file and other files as well, so the menu.rb file became with no use. Try remove it and run the app, and it will work fine.

Reply:
Please don't forget that there are is sub-menus as well, so it would be a nice code structure to leave a main entry point for the main menu separate in a single file.